rossum_api/api_client.py

```diff
@@ -427,13 +423,34 @@ async def get_token(self, refresh: bool = False) -> str:
         return self.token  # type: ignore[return-value] # self.token is set in _authenticate method
 
     async def _authenticate(self) -> None:
-        response = await self.client.post(
-            f"{self.base_url}/auth/login",
-            data={"username": self.username, "password": self.password},
-        )
-        await self._raise_for_status(response)
+        async for attempt in self._retrying():
+            with attempt:
+                response = await self.client.post(
+                    f"{self.base_url}/auth/login",
+                    data={"username": self.username, "password": self.password},
+                )
+                await self._raise_for_status(response)
         self.token = response.json()["key"]
 
+    def _retrying(self):
+        """Build Tenacity retrying according to desired settings."""
+
+        def should_retry_request(exc: BaseException) -> bool:
+            if isinstance(exc, httpx.RequestError):
+                return True
+            if isinstance(exc, httpx.HTTPStatusError):
+                return exc.response.status_code in RETRIED_HTTP_CODES
```

Comment:
401 is not in RETRIED_HTTP_CODES so we can re-use this retrying config also for authentication.